src/components/FeedbackReportForm/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './FeedbackReportForm'
```

Comment:
I've recently noticed that you started using this. Is it simply to have a more descriptive file name than `index.tsx` for the component to work in?

Reply:
Exactly and it makes searching the file easier!